src/class_resolver/base.py

```diff
@@ -251,7 +251,7 @@ def get_option(
         # TODO are there better ways to type options?
         return click.option(  # type:ignore
             *flags,
-            type=click.Choice(list(self.lookup_dict), case_sensitive=False),
+            type=click.Choice(list(self.options), case_sensitive=False),
```

Comment:
Not decided about this. Won’t having more options be confusing? On the other hand, there might be non-trivial synonyms. Have you run into this case?

Reply:
This came up while working on https://github.com/pykeen/pykeen/pull/1452, where the options for the similarity are `NegativeKullbackLeiblerDivergence` (synonym: `KL`) and `ExpectedLikelihood` (synonym: `EL`). Especially the former is quite long to type on the command line.

Reply:
Okay. Then maybe we should make our own class that’s like click.Choice that gives a bit of context to the options (maybe it’s hack able to change the way the help text is written and we can directly subclass it)